validator.js

```diff
@@ -31,39 +36,40 @@ function getValidatorStats() {
                 let errors = await errorChecker.checkForErrors(validator, CONFIG, response)
                 //validator = response.account_view.address
                 calNotification(validator, errors)
+                if (errors.length > 0) {
```

Comment:
Instead of checking for error here, why not just add a branching path in sendNotification()?
If an error exists and is about to be pushed to other channels adding another channel (email) there seems to be a better choice than seperate it here.